Documentation/data-collection.md

```diff
@@ -21,318 +37,495 @@ data:
     - '{__name__="cluster_legacy_scheduler_policy"}'
     #
-    # (workloads-team, @openshift/openshift-group-b) cluster_master_schedulable reports whether mastersSchedulable=true in
+    # (openshift/openshift-team-master, @openshift/openshift-group-b)
+    #
+    # cluster_master_schedulable reports whether mastersSchedulable=true in
     # the scheduler operator. This value is a boolean 0|1
     - '{__name__="cluster_master_schedulable"}'
     #
-    # (codeready workspaces, @ibuziuk) The number of workspaces with a given status STARTING|STOPPED|RUNNING|STOPPING. Type 'gauge'.
+    # (codeready workspaces, @ibuziuk)
+    # The number of workspaces with a given status STARTING|STOPPED|RUNNING|STOPPING. Type 'gauge'.
     - '{__name__="che_workspace_status"}'
     #
-    # (codeready workspaces, @ibuziuk) The number of started workspaces. Type 'counter'.
+    # (codeready workspaces, @ibuziuk)
+    # The number of started workspaces. Type 'counter'.
     - '{__name__="che_workspace_started_total"}'
     #
-    # (codeready workspaces, @ibuziuk) The number of failed workspaces.
+    # (codeready workspaces, @ibuziuk)
+    # The number of failed workspaces.
     # Can be used with the 'while' label e.g. {while="STARTING"}, {while="RUNNING"}, {while="STOPPING"}.Type 'counter'.
     - '{__name__="che_workspace_failure_total"}'
     #
-    # (codeready workspaces, @ibuziuk) The time in seconds required for the startup of all the workspaces.
+    # (https://github.com/redhat-developer/codeready-workspaces, @ibuziuk)
+    #
+    # The time in seconds required for the startup of all the workspaces.
     - '{__name__="che_workspace_start_time_seconds_sum"}'
     #
-    # (codeready workspaces, @ibuziuk) The overall number of attempts for starting all the workspaces.
+    # (https://github.com/redhat-developer/codeready-workspaces, @ibuziuk)
+    #
+    # The overall number of attempts for starting all the workspaces.
     - '{__name__="che_workspace_start_time_seconds_count"}'
     #
-    # (cloud credential operator, @openshift/openshift-team-hive) Track current mode the cloud-credentials-operator is functioning under.
+    # (@openshift/openshift-team-hive)
+    #
+    # Track current mode the cloud-credentials-operator is functioning under.
     - '{__name__="cco_credentials_mode"}'
     #
-    # (@openshift/storage, OpenShift Storage team) Persistent Volume usage metrics: this is the number of volumes per plugin
+    # (@openshift/storage, OpenShift Storage team)
+    #
+    # Persistent Volume usage metrics: this is the number of volumes per plugin
     # and per volume type (filesystem/block)
     - '{__name__="cluster:kube_persistentvolume_plugin_type_counts:sum"}'
     #
-    # (Advanced Cluster Management, @open-cluster-management/squad-kui-admins) visual_web_terminal_sessions_total is the count of Visual Web Terminal sessions created 
+    # (Advanced Cluster Management, @open-cluster-management/squad-kui-admins)
```

Comment:
folks would need to be in the open-cluster-management org to be able to reach out to them. Maybe we should point out to a forum instead if an openshift team is not possible.